go/tests/prog/grouping_prog/main.go

```diff
@@ -45,17 +45,16 @@ func brackets(regexp, text string) {
 		runes = runes[1 : len(runes)-1]
 		result := simpleSearch(runes, text)
 		for i, s := range result {
-			if !unicode.Is(unicode.Hex_Digit, rune(s[len(s)-1])) {
-				s = s[:len(s)-1]
+			if !unicode.Is(unicode.Hex_Digit, rune(s[len(s)-1])) && !unicode.IsLetter(rune(s[len(s)-1])) {
```

Comment:
if
```go
!unicode.IsLetter(rune(s[len(s)-1]))
```
is true then
```go
!unicode.Is(unicode.Hex_Digit, rune(s[len(s)-1]))
```
is always true

Reply:
I removed one of the ifs that were repeated and tested the exercise several times according to the readme and the tests and everything is working now

Reply:
Yes it is working but what I meant is that those conditions are certainly similar :

```go
if !unicode.Is(unicode.Hex_Digit, rune(s[len(s)-1])) && !unicode.IsLetter(rune(s[len(s)-1])) {
if !unicode.IsLetter(rune(s[len(s)-1])) {
```

but the latter is simpler, don't you think ?

Reply:
ohhh, yes you are right, i tested it only with the first condition and it didn't work, but if we do it with the second one only it works fine, my bad... 
do you think we need to change this?

Reply:
Yes it is important to simplify code ^^ it makes maintenance and reading easier. You can modify the line, add/commit/push on master !